README.md

```diff
@@ -402,12 +402,15 @@ window._zx.push({"id": "25GHTE9A07DF67DFG90T"});
 
 #### Conversion tracking
 
+Lead events and sale events use similar text snippets, but different url paths. For example:
+`https://foo.zanox.com/abc/?123456789`, where `abc` is the changing url path. This can be set dynamically along with the rest of the parameters by setting the `path_extension`. 
+
 This is an example of a lead event:
 
 ```
 def show
   tracker do |t|
-    t.zanox :track, { order_i_d: 'DEFC-4321'}
+    t.zanox :track, { order_i_d: 'DEFC-4321', path_extension: 'abc' }
```

Comment:
it's a matter of taste:
```
t.zanox :lead, { order_i_d: 'DEFC-4321' }
t.zanox :sale, { order_i_d: 'DEFC-4321' }
```
vs
```
t.zanox :track, { order_i_d: 'DEFC-4321', type: 'lead' }
t.zanox :track, { order_i_d: 'DEFC-4321', type: 'sale' }
```
the former is to me clearer, and the implementation would be easier the path can be set in the `write` method of each event.